src/pages/List.js

```diff
@@ -55,11 +51,27 @@ export default function List({ token }) {
   };
 
   function compareTimeStamps(lastPurchased) {
-    const currentElapsedMilliseconds = Date.now();
-    const millisecondsInOneDay = 86400000;
-    return currentElapsedMilliseconds - lastPurchased < millisecondsInOneDay;
+    const currentTimestamp = Math.floor(Date.now() / 86400000);
+    return currentTimestamp - lastPurchased < 1;
   }
 
+  const alphabetizeListItems = (list) => {
+    const sortedList = list.sort((a, b) => {
+      if (a.data().item_name < b.data().item_name) {
+        return -1;
+      }
+      if (a.data().item_name > b.data().item_name) {
+        return 1;
+      }
+      return 0;
+    });
+    return sortedList;
+  };
+
+  const checkForInactiveItem = (item) =>
+    Math.floor(Date.now() / 86400000) - item.data().last_purchased <
+    item.data().last_estimate;
+
```

Comment:
What does this function check for? 

AC described Inactive as: 
> Inactive (when there’s only 1 purchase in the database or the purchase is really out of date [the time that has elapsed since the last purchase is 2x what was estimated])

Inactive is true if:
- `itemData.times_purchased <= 1 ` OR
- time elapsed since last purchase date >=  two times `last_estimate`

If the `last_estimate` for milk is 5 days, and it's been 14 days since I bought milk, is it considered inactive?

Reply:
Has this been resolved @jssckbl @jamesncox?